com.unity.render-pipelines.universal/ShaderLibrary/Unlit.hlsl

```diff
@@ -0,0 +1,50 @@
+
+#ifndef UNLIT_INCLUDED
+#define UNLIT_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Debug/Debugging3D.hlsl"
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl"
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/SurfaceData.hlsl"
+
+half4 UniversalFragmentUnlit(InputData inputData, SurfaceData surfaceData)
+{
+    half3 albedo = surfaceData.albedo;
+
+    #if defined(_ALPHAPREMULTIPLY_ON)
+    albedo *= surfaceData.alpha;
+    #endif
+
+    #if defined(DEBUG_DISPLAY)
+    half4 debugColor;
+
+    if (CanDebugOverrideOutputColor(inputData, surfaceData, debugColor))
+    {
+        return debugColor;
+    }
+    #endif
+
+    half4 finalColor = half4(albedo + surfaceData.emission, surfaceData.alpha);
+
+    return finalColor;
+}
+
+// TODO: Legacy code - is it safe to remove this?
```

Comment:
As follow up should just update this comment to say this is deprecated.

Reply:
Done